SU-HSDO/suhumsci
# Feat(STN-835): spotlight slider styles

# READY FOR REVIEW

## Summary
Utilized slider mixins to implement styles for Spotlight Slider. 
No new styles were added.

## Need Review By (Date)
asap

## Designs
[figma](https://www.figma.com/file/BieDQhj8BoLlHJrzpDaIXV/Slider-Container-for-Hero-Components?node-id=0%3A1)

## Urgency
medium

## Steps to Test
1. Create a new Content Page and utilizing the newly added Spotlight Slider and confirm designs match figma (both full-width and inside Well)
2. Add Gradient Hero Slider component and confirm its styles haven't changed.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
